src/passwordHelpers.js

```diff
@@ -0,0 +1,17 @@
+import bcrypt from 'bcrypt-nodejs';
+
+export function comparePass(password, user) {
+  return new Promise((resolve, reject) => {
+    bcrypt.compare(password, user.password, (error, isAuth) => {
+      isAuth ? resolve(isAuth) : reject(error);
+    });
+  });
+}
+
+export function hashPass(password) {
+  return new Promise((resolve, reject) => {
+    bcrypt.hash(password, null, null, (error, hash) => {
```

Comment:
Why not use a salt? We use the id of a user appended to a random string to make the hashes better